goshopify.go

```diff
@@ -368,6 +379,40 @@ func (c *Client) doGetHeaders(req *http.Request, v interface{}) (http.Header, er
 	return resp.Header, nil
 }
 
+func (c *Client) logRequest(req *http.Request) {
+	if req == nil {
+		return
+	}
+	if req.URL != nil {
+		c.log.Debugf("%s: %s", req.Method, req.URL)
+	}
+	if req.Body != nil {
```

Comment:
This block is the same as the block starting line 405 - consolidate and move into a function `logBody` ?

Reply:
also a great idea!